intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_utils.cpp

```diff
@@ -106,6 +115,8 @@ static bool kv_cache_init(const struct model_hparams& hparams, struct model_kv_c
       ne_set_name(k_cache, "cache_k");
       ne_set_name(v_cache, "cache_v");
     }
+    const bool run_mha_reordered = model->layers[0].k_cache->type == NE_TYPE_JBLAS;
+    fprintf(stderr, "%s: run_mha_reordered = %d\n", __func__, run_mha_reordered);
```

Comment:
Debug?

Reply:
also for inference. It shows the MHA fusion clearly.